modmesh/apputil.py

```diff
@@ -88,14 +114,17 @@ def get_appenv(name=None):
 get_appenv(name='master')
 
 
-def run_code(code):
+def run_code(code, redirectStdOutFd=-1, redirectStdErrFd=-1):
```

Comment:
In my imagination, the code handling fd would be more straight-forward if they are in C/C++.  In Python, the fd should be wrapped by the Python file object, which is used almost everywhere in Python.